pages/management/devices.md

```diff
@@ -32,6 +34,7 @@ An extremely useful feature of the devices detail page is the log window. This i
 
 On the device's Actions page we give you access to a number of useful per-device actions.
 
+<!-- NOTE: is it really necessary to reload the page while the ID is located on the same page?-->
```

Comment:
I was just wondering if it was necessary to link in the fashion of `/management/#idtoheading` instead of just `#idtoheading` since the former triggers a full page reload in the browser where as the latter only navigates to the ID on the page without actually reloading the URL.